root/components/forms.js

```diff
@@ -99,6 +99,10 @@ function subfieldErrors(field, accum = Immutable.List()) {
   return accum;
 }
 
+const makeErrorListItem = (error, index) => (
```

Comment:
In cases where the `map` callback depends only the array items, I factored out the callback function to avoid creating a new function/closure on each render.

Reply:
Just a note -  we used buildFoo in the headers PR.